study.md

```diff
@@ -29,21 +29,25 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+The event loop is the the management of tasks a program executes that, when illustrated, has a loop like
+nature. the loop puts callback functions in a task que and when the task in the stack is complete, the task in the cue
+goes to the stack. When that is implemented the next task in the que gets pushed to the stack. So on and so forth..
```

Comment:
Super close here. The call stack is where synchronous code waits to be ran. Asynchronous code wait in the queue. The event loop waits for the stack to be empty, then it grabs the next thing waiting in the queue and moves it to the stack to be run.